net-dialup/linux-atm/linux-atm-2.5.2-r2.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic linux-info
+
+DESCRIPTION="Tools for ATM"
+HOMEPAGE="https://linux-atm.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Any specific reason for dropping all keywords here?

Reply:
Because I haven't tested if it even builds on those other platforms.

Reply:
Please add them back and just destabilize everything (`ekeyword ~all`).

We don't drop keywords so lightly, fixing a few modern C issues has a very low chance of breaking anything.  Please read more about this [here](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades).